packages/reducers/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@redux-tools/injectors": "^0.8.2",
     "@redux-tools/namespaces": "^0.8.2",
+    "@redux-tools/namespaces-react": "^0.8.2",
```

Comment:
nonono, `reducers` cannot depend on `namespaces-react`, `reducers` is platform-agnostic :) please remove